### PR TITLE
Edit  Chapter 1: Object Basics and ask question in TODO

### PR DESCRIPTION
**Pull request by Clay-Mysten**
Make many small edits to programming objects Chapter 1: Object Basics
Add TODO for clarity on sui repo install

### CODE REVIEW

File: doc/src/build/programming-with-objects/ch1-object-basics.md [hunk 1]
```diff
@@ -71,30 +71,34 @@ public fun create(red: u8, green: u8, blue: u8, ctx: &mut TxContext) {
...
 
-To compile the code, make sure you have followed the [Sui installation guide](../install.md) so that `sui-move` is in `PATH`. In the code [root directory](../../../move_code/objects_tutorial/) (where `Move.toml` is), you can run:
+To compile the code, make sure you have [installed Sui](../install.md) so that `sui-move` is in `PATH`. In the code [root directory](../../../move_code/objects_tutorial/) (where `Move.toml` is), run:
 ```
 sui-move build
 ```
 
-### Writing Unit Tests
-After defining the `create` function, we want to test this function in Move using unit tests, without having to go all the way to sending Sui transactions. Since Sui manages global storage separately outside of Move, there is no direct way to retrieve objects from global storage within Move. This poses a question: after calling the `create` function, how do we check that the object is properly transferred?
+TODO: Doesn't this assume they have downloaded the entire `sui` repo too? `install.md` gets them the `sui-move` binary but not the repo. Adam asked me to keep that out of install and so instead lives at [Contribute to Sui](../../contribute/index.md) with `git clone` covered there.
```
Comment (lxfind) [inline doc/src/build/programming-with-objects/ch1-object-basics.md:88]: You are right, this does require the Move files used in the tutorial available locally. For now I guess it means they have to clone the repo.
But that doesn't sound ideal.
I can see two possible solutions:
1. Put the tutorial and files in a separate repository, which they can clone separately
2. I wonder if there is a way to also download these tutorial files when using git install

Comment (Clay-Mysten) [reply doc/src/build/programming-with-objects/ch1-object-basics.md:88]: I think they should sync the repo. I am proposing in Slack we simply make download/clone an optional section in Install. We already link to the Contribute page from there for the step. But that's a circuitous journey for our users.

**Review by lxfind — APPROVED** [review]

File: doc/src/build/programming-with-objects/ch1-object-basics.md (hunk 1 above)
Comment (Clay-Mysten) [reply doc/src/build/programming-with-objects/ch1-object-basics.md]: Taking care of this in:
https://github.com/MystenLabs/sui/pull/1374
https://github.com/MystenLabs/sui/pull/1375

**Comment by Clay-Mysten** [comment]
And this gets the installation into prerequisites:
https://github.com/MystenLabs/sui/pull/1376

All done with this PR.

**Pull request merged**